pkg/apis/stork/v1alpha1/applicationclone.go

```diff
@@ -0,0 +1,86 @@
+package v1alpha1
+
+import (
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	//ApplicationCloneResourceName is the name for the application clone resource
+	ApplicationCloneResourceName = "applicationclone"
+	//ApplicationCloneResourcePlural is the name in plural for the application clone resources
+	ApplicationCloneResourcePlural = "applicationclones"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+//ApplicationClone represents the cloning of application in different namespaces
+type ApplicationClone struct {
```

Comment:
Also need to update `register.go` in this directory to add `ApplicationClone` and `ApplicationCloneList` to the schema